src/components/Sidebar/Sidebar.js

```diff
@@ -124,35 +125,46 @@ class Sidebar extends PureComponent<Props, State> {
         {({ sidebarOffsetPercentage, firstProjectPosition }) => (
           <Fragment>
             <Wrapper offset={`${sidebarOffsetPercentage}%`}>
-              <IntroductionBlurb
-                isVisible={!finishedOnboarding && introSequenceStepIndex >= 1}
-              />
-
-              <Projects offset={`${firstProjectPosition}px`}>
-                {projects.map(project => (
-                  <Fragment key={project.id}>
-                    <Tooltip title={project.name} position="right">
-                      <SidebarProjectIcon
-                        size={SIDEBAR_ICON_SIZE}
-                        id={project.id}
-                        name={project.name}
-                        color={project.color}
-                        iconSrc={project.icon}
-                        isSelected={
-                          finishedOnboarding && project.id === selectedProjectId
-                        }
-                        handleSelect={() => selectProject(project.id)}
-                      />
-                    </Tooltip>
-                    <Spacer size={18} />
-                  </Fragment>
-                ))}
-                <AddProjectButton
-                  size={SIDEBAR_ICON_SIZE}
-                  onClick={createNewProjectStart}
-                  isVisible={finishedOnboarding || introSequenceStepIndex >= 2}
+              <ScrollbarOnlyVertical
+                autoHide
+                hideTracksWhenNotNeeded
+                renderTrackHorizontal={props => (
+                  <div {...props} style={{ display: 'none' }} />
```

Comment:
OK, this is working but I'm wondering if there is a better way but I couldn't find anything.
I think it would be better to have the width right so that `react-custom-scrollbars` doesn't render the horizontal bar.

I also played a bit with `renderTrackHorizontal` and I don't get the expected result e.g. pass the styles donw to the div and add a `backgroundColor` like mentioned in the docs - not sure why that's not working.
For me it hides the scrollbar as soon I'm adding `renderTrackHorizontal` prop.

Anyway, if there is no other way it's OK to leave it as it is now.

Reply:
I felt the same when doing this, feels hacky but didnt found any workaround (this took 80% of the PR 🤣), if you preffer I can only add the `renderTrackHorizontal` prop. Or maybe make a sort of wrapper component name `VerticalScrollbar`?